src/app/app-routing.module.ts

```diff
@@ -16,16 +16,20 @@ import { HistorySubviewComponent } from './components/views/job-view/subviews/hi
 import { FacilitiesSubviewComponent } from './components/views/job-view/subviews/facilities-subview/facilities-subview.component';
 import { TestsSubviewComponent } from './components/views/job-view/subviews/tests-subview/tests-subview.component';
 import { WeatherViewComponent } from './components/views/weather-view/weather-view.component';
+import { LoginViewComponent } from './components/views/login-view/login-view.component';
+import { ProfessorComponent } from './components/views/login-view/professor/professor.component';
 
 const routes: Routes = [
-  { path: '', component: HomepageComponent, pathMatch: 'full' },
+  { path: '', component: LoginViewComponent, pathMatch: 'full' },
```

Comment:
We should talk about the routes. You can skip the login page by opening any other page and then clicking the globe. Like what do we want to be the '' page. We could possibly implement either route protection (def need to rewatch those vids, don't remember that at all) or maybe a redirect built into a high level of the component structure that will redirect you to the login if you're not logged in.